Reed/Reader/views/ReaderView.swift

```diff
@@ -42,12 +50,11 @@ struct ReaderView: View {
                     width: UIScreen.main.bounds.width,
                     height: UIScreen.main.bounds.height * 0.65
                 ))
-                Text("\(viewModel.curPage + 1) of \(viewModel.pages.endIndex)")
+                Text(viewModel.getPageNumberDisplay())
```

Comment:
Does this continue to show the right page even after flipping? If not we'll have to use a binding.

Reply:
it continues to show the right page